pkgs/applications/science/misc/root/default.nix

```diff
@@ -151,40 +151,18 @@ stdenv.mkDerivation rec {
   '';
 
   cmakeFlags = [
-    "-Drpath=ON"
     "-DCMAKE_INSTALL_BINDIR=bin"
     "-DCMAKE_INSTALL_LIBDIR=lib"
     "-DCMAKE_INSTALL_INCLUDEDIR=include"
     "-Dbuiltin_llvm=OFF"
-    "-Dbuiltin_freetype=OFF"
-    "-Dbuiltin_gtest=OFF"
-    "-Dbuiltin_nlohmannjson=OFF"
-    "-Dbuiltin_openui5=ON"
-    "-Dclad=ON"
-    "-Ddavix=ON"
-    "-Ddcache=OFF"
     "-Dfail-on-missing=ON"
-    "-Dfftw3=OFF"
     "-Dfitsio=OFF"
-    "-Dfortran=OFF"
     "-Dgnuinstall=ON"
-    "-Dimt=ON"
-    "-Dgviz=OFF"
-    "-Dhttp=ON"
     "-Dmysql=OFF"
-    "-Dodbc=OFF"
```

Comment:
We don't want to rely on autoconfiguration here as well. If Nix build is running outside of sandbox, we don't want to let it pick up some dependencies from the system and fail the build.

Reply:
See https://github.com/NixOS/nixpkgs/pull/308497#discussion_r1588474977